ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.scss

```diff
@@ -0,0 +1,391 @@
+        background-position: bottom;
+        background-size: contain;
+
+        .title {
+            margin-top: 0;
+            margin-bottom: 16px;
+        }
+
+        .logos {
+            img:first-child {
+                margin-right: 5rem;
+            }
+        }
+    }
+
+    &.learn-more {
+        padding: 80px 0 65px;
+
+        .title {
+            margin-bottom: 32px;
+        }
+    }
+
+    &.join {
+        height: 470px;
+        padding: 36px 40px 55px 100px;
+        display: flex;
+        box-sizing: border-box;
+
+        .image {
+            flex: 0 1 400px;
+            margin-right: 100px;
+            background-image: url(/assets/images/map.png);
+            background-repeat: no-repeat;
+            background-position: top;
+            background-size: contain;
+        }
+
+        .description {
+            flex: 1 2 auto;
+            padding-top: 70px;
+        }
+
+        .title {
+            margin-bottom: 34px;
+        }
+        .text {
+            text-align: left;
+
+            &:first-of-type {
+                margin-bottom: 32px;
+            }
+        }
+    }
+
+    &.participate {
+        padding: 112px 20px;
+        text-align: left;
+
+        .title {
+            margin-bottom: 78px;
+        }
+
+        .container {
+            max-width: $container-width;
+            margin: 0 auto;
+            color: #000000;
+
+            &.steps {
+                display: flex;
+                justify-content: space-between;
+            }
+        }
+
+        .step {
+            margin: 0 1.5rem 0 0;
+            font-family: Untitled-Sans-Regular;
+            font-size: 16px;
+
+            &:last-child {
+                margin-right: 0;
+            }
+
+            &__image {
+                width: 150px;
+                height: 150px;
+                display: block;
+                margin: 0 auto 26px;
+                position: relative;
+                left: -10%;
+            }
+
+            &__title {
+                display: inline;
+                font-family: Untitled-Sans-Medium;
+                letter-spacing: -0.07px;
+                line-height: 24px;
+                margin-right: 8px;
+            }
+
+            &__time {
+                letter-spacing: -0.07px;
+                line-height: 24px;
+            }
+
+            &__description {
+                font-size: 24px;
+                letter-spacing: -0.1px;
+                line-height: 28px;
+                margin-top: 16px;
+                max-width: 240px;
+            }
+        }
+    }
+
+    &.faq {
+        padding: 110px 0 60px 0;
+
+        .container {
+            box-sizing: border-box;
+            max-width: 900px;
+            padding: 0 60px;
+            margin: 56px auto;
+            text-align: left;
+        }
+    }
+
+    &.stay-informed {
+        padding: 86px 20px 72px 20px;
+
+        .container {
+            box-sizing: border-box;
```

Comment:
do we need it?

Reply:
Not exactly. Seems to be copied from mockup. 
Removed at c33f7af1b2894ff2743ac4cca7ab2af0a7664535